source/agora/test/Byzantine.d

```diff
@@ -225,7 +226,7 @@ private class ByzantineManager (bool addSpyValidator = false,
     }
 }
 
-/// Block should be added if we have 5 / 6 validators signing (our SpyValidator does not sign)
+/// Block should be added if we have 6 / 6 validators signing
```

Comment:
Isn't it still 5 out of 6?

Reply:
The SpyValidator now also signs so it is 6/6 but I will update the threshold to match this.